tests/e2e/wdio.update.conf.js

```diff
@@ -138,10 +137,16 @@ export const config = {
           ),
         );
 
+        // Ensure extension reloaded the source, so we can open the settings page
+        await browser.pause(5000);
```

Comment:
maybe this can be replaced with some `browser.waitUntil` and some message passing? we can await response from background to be sure it is ready. Most of the time this should be faster than 5 seconds and in case it will be slower the tests will always pass. (but you may want to extend a default mocha timeout for this step)